packages/storybook/vite.config.js

```diff
@@ -11,6 +11,17 @@ import ts from 'vite-plugin-ts';
 
 // https://vitejs.dev/config/
 export default defineConfig({
+  base: './',
+  plugins: [ts()],
+  optimizeDeps: {
+    esbuildOptions: {
+      tsconfigRaw: {
+        compilerOptions: {
+          experimentalDecorators: true,
```

Comment:
Need this to avoid errors like the following on `yarn watch:vite` when using `outline-core-link`.

```
✘ [ERROR] Transforming JavaScript decorators to the configured target environment ("chrome87", "edge88", "es2020", "firefox78", "safari14" + 2 overrides) is not supported yet

    node_modules/@phase2/outline-core-link/src/outline-core-link.ts:49:2:
      49 │   @property({ type: String, attribute: 'link-href' })
         ╵   ^
```